src/module/entities/helper.js

```diff
@@ -246,7 +246,7 @@ module.exports = class UserProjectsHelper {
 					status: CONSTANTS.common.ACTIVE_STATUS,
 				}
 				// Specify the fields to include in the result set
-				const userRoleExtensionProjection = ['_id', 'title', 'userRoleId', 'userType']
+				const userRoleExtensionProjection = ['_id', 'label', 'value']
```

Comment:
@MallanagoudaB revert the change. after you get the data you have to write logic to trnasform the response to the required form